src/Aspire.Dashboard/Components/Controls/GridValue.razor

```diff
@@ -44,15 +44,18 @@
         ];
     }
 
-    <div @onclick:stopPropagation="true">
-        <FluentButton Appearance="Appearance.Lightweight"
-                      Id="@_anchorId"
-                      Class="defaultHidden"
-                      Style="float: right; flex-shrink: 0"
-                      AdditionalAttributes="@FluentUIExtensions.GetClipboardCopyAdditionalAttributes(ValueToCopy ?? Value, PreCopyToolTip, PostCopyToolTip, uncapturedAttributes)">
-            <FluentIcon Class="copy-icon" Style="display:inline;" Icon="Icons.Regular.Size16.Copy" />
-            <FluentIcon Class="checkmark-icon" Style="display:none;" Icon="Icons.Regular.Size16.Checkmark" />
-        </FluentButton>
-    </div>
-    <FluentTooltip @ref="_tooltipComponent" Anchor="@_anchorId" Position="TooltipPosition.Top" HideTooltipOnCursorLeave="true">@PreCopyToolTip</FluentTooltip>
+    @if (!ViewportInformation.IsUltraLowWidth)
```

Comment:
> Oh I see in your PR description:
> 
> > but since this is a nonessential feature also available by double-clicking the cell, I removed the copy button on very low viewport widths to allow for more content to be displayed.
> 
> Is double-clicking a thing on mobile devices?

Selecting all would have been a better choice of words for me, which is implemented